### PR TITLE
Add test coverage

### PR DESCRIPTION
**Pull request by ryanmitchell**
This PR adds test coverage for the add-on.

- [x] Test tags
- [x] Test [webhook urls](https://statamic-shopify-docs.vercel.app/CMS/webhooks)
- [x] Test [add-on 'action' urls](https://statamic-shopify-docs.vercel.app/CMS/actions) 
- [x] Add test Github action

### CODE REVIEW

File: docs/content/en/CMS/actions.md
Comment (edalzell) [inline docs/content/en/CMS/actions.md:1]: why is a docs change in a test PR?

File: phpunit.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  </testsuites>
+  <php>
+    <env name="APP_ENV" value="testing"/>
+    <env name="APP_KEY" value="base64:xRIcDp1ReW8Y8rd9V9D7hOVV4TI7ThCF3FKxRg01Rm8="/>
+    <env name="CACHE_DRIVER" value="array"/>
+    <env name="DB_CONNECTION" value="testing"/>
+    <env name="SESSION_DRIVER" value="array"/>
+    <env name="QUEUE_DRIVER" value="sync"/>
+    <env name="MAIL_DRIVER" value="array"/>
+    <env name="MAILCHIMP_DRIVER" value="log"/>
```
Comment (edalzell) [inline phpunit.xml:20]: why is this line needed?

File: src/Http/Controllers/Actions/VariantsController.php
Comment (edalzell) [inline src/Http/Controllers/Actions/VariantsController.php:1]: why is this change here?

File: docs/content/en/CMS/actions.md
Comment (ryanmitchell) [reply docs/content/en/CMS/actions.md:1]: I changed the method response when working on the tests. Too many overlapping PRs right now.

Comment (edalzell) [reply docs/content/en/CMS/actions.md:1]: we can always have an order to merging, to keep the PRs smaller, as you can do the merging.

File: src/Http/Controllers/Actions/VariantsController.php [hunk 2]
```diff
@@ -10,19 +10,11 @@ class VariantsController extends CpController
 {
     public function fetch(Request $request, $product)
     {
```
Comment (edalzell) [inline src/Http/Controllers/Actions/VariantsController.php:12]: typehint?
